### DISCUSSION TITLE
Sidebar component

### DISCUSSION THREAD

**Discussion by adolfusadams** [unovue/shadcn-vue#823] [Ideas]
Shadcn released an awesome new **Sidebar** component. See: https://ui.shadcn.com/docs/components/sidebar. When would this **Sidebar** component be added to Shadcn-vue?

**Comment by MrGaude** [comment]
100% agree this should be included. Currently I'm  just running with one I made myself.

**Comment by brokuka** [comment]
Component already [added](https://www.shadcn-vue.com/docs/components/sidebar.html)
